libraries/classes/Util.php

```diff
@@ -1756,6 +1757,16 @@ public static function unsupportedDatatypes(): array
         return [];
     }
 
+    /**
+     * This function is to check whether database support UUID
+     *
+     * @return bool
+     */
+    public static function isUUIDSupported()
```

Comment:
```suggestion
    public static function isUUIDSupported(): bool
```
phpdoc return can be removed

Reply:
updated. thank you ^_^